pkg/api/customization/clusterscan/clusterscan.go

```diff
@@ -21,7 +21,11 @@ const (
 
 func Formatter(apiContext *types.APIContext, resource *types.RawResource) {
 	if err := apiContext.AccessControl.CanDo(mgmtv3.ClusterScanGroupVersionKind.Group, mgmtv3.ClusterScanResource.Name, "read", apiContext, resource.Values, apiContext.Schema); err == nil {
-		status := resource.Values["status"].(map[string]interface{})
+		s, ok := resource.Values["status"]
+		if !ok {
+			return
+		}
+		status := s.(map[string]interface{})
```

Comment:
nit: we also have convert.ToMapInterface

Reply:
👍 will change